spec/win32/screenshot/take_spec.rb

```diff
@@ -19,8 +19,8 @@
   it "captures an area of the foreground" do
     image = Win32::Screenshot::Take.of(:foreground, :area => [30, 30, 100, 150])
     save_and_verify_image(image, 'foreground_area')
-    expect(image.width).to eq(70)
-    expect(image.height).to eq(120)
+    expect(image.width).to eq(100)
```

Comment:
I believe the culprit are that these lines from the original [capture_area](https://github.com/jarmo/win32screenshot/blob/88e209986d1de09f5afff0d1f4a9bcfbdf281db6/lib/win32/screenshot/bitmap_maker.rb#L54) function is no longer there.

I'll try to figure out a better solution for my current implementation

Reply:
Maybe since your additions capture the whole window (even not just the visible area, as was done before), then it makes sense to add a separate :type to the `Take#of` method to preserve backwards compatibility? Maybe something in the lines of `:window_full` or something to hint that it is different from `:window`?

I'm pretty sure that there exists code where `:area` has been set and when `x1` and/or `y1` are different from 0 then behavior will change with these changes (and even having these options do not add any value when they are not used).

It would be great though to detect somehow if capturing a screenshot with `:window` is even possible or if black screen will be captured to throw an error and suggest using `:window_full` or whatever. Not sure if it's possible.

Now that I'm thinking about it when writing this comment I'm starting to think that maybe backwards-incompatibility should be introduced and `:area` option should be removed altogether - let's just try to make full screenshot all the time from whatever is asked (either `:desktop`, `:foreground` or `:window`) and if anyone cares only about the area then image postprocessing should be done separately by some other library/tool. In that way `win32screenshot` does only one thing - takes screenshots.

What are your thoughts?

Reply:
I think that makes a lot of sense. As it is right now, our team is only interested in capturing windows/whole desktop. Maybe I will work on something like specifying primary monitor when capturing desktop.

I'll work on committing some code that removes the area functionality all together. Thanks for your cooperation

Reply:
Please also remove mentions of `area` options in the documentation of `Take#of`.